astropy/modeling/functional_models.py

```diff
@@ -365,6 +370,13 @@ def __init__(self, amplitude=amplitude.default, x_mean=x_mean.default,
                 y_vec = eig_vecs[:, 0]
                 theta = np.arctan2(y_vec[1], y_vec[0])
 
+        # Ensure stddev makes sense if its bounds are not explicitly set.
+        # stddev must be non-zero and positive.
+        # Setting this in Parameter above causes convolution tests to hang.
```

Comment:
I would add a TODO or FIXME here - this is clearly something that should be fixed in future.